bilge-impl/src/from_bits.rs

```diff
@@ -79,7 +47,8 @@ fn generate_struct(arb_int: TokenStream, struct_type: &Ident) -> TokenStream {
     }
 }
 
-fn generate_common(expanded: TokenStream, type_name: &Ident) -> TokenStream {
+fn generate_common(expanded: TokenStream, type_name: &Ident, _reserved_variant: Option<Variant>) -> TokenStream {
+    // TODO: if reserved_variant.is_some(), an assert should not be generated
```

Comment:
Hmm, I don't think it's that easy. Just now I _thought_ we just need to note that `FILLED` is not respecting `#[reserved]` fields, but we actually need it to respect them since structs need to know if their fields are filled.
So, you have to just get the reserved variants here as well:
https://github.com/hecatia-elegua/bilge/blob/f59d6600f59195a8240752f0ed32823167a4e4d9/bilge-impl/src/bitsize.rs#L238